Tribler/Core/simpledefs.py

```diff
@@ -47,6 +47,7 @@
 STATEDIR_SESSCONFIG = 'libtribler.conf'
 STATEDIR_DLCONFIG = 'tribler.conf'
 STATEDIR_GUICONFIG = 'tribler.conf'
+CONFIGFILE_NAME = 'libtribler.ini'
```

Comment:
I'd keep the extension and constant convention. What about `STATEDIR_CONFIG = 'triblerd.conf'`? Once the wx gui gets removed we can reuse `STATEDIR_GUICONFIG` for the new one and remove the rest.